src/Common/AsynchronousMetrics.h

```diff
@@ -64,7 +64,7 @@ class AsynchronousMetrics
     using ProtocolServerMetricsFunc = std::function<std::vector<ProtocolServerMetrics>()>;
 
     AsynchronousMetrics(
-        int update_period_seconds,
+        unsigned update_period_seconds,
```

Comment:
In general I like this change, because negative values don't make sense here, and also the regarding setting is an `UInt32`, however now there is a difference between `AsynchronousMetrics` and `ServerAsynchronousMetrics`/`KeeperAsynchronousMetrics`. I would say either change all of them, or none of them to keep consistency.